src/core/solvers/ddp.cpp

```diff
@@ -279,9 +293,11 @@ void SolverDDP::backwardPass() {
       throw_pretty("backward_error");
     }
   }
+  STOP_PROFILER("SolverDDP::backwardPass");
 }
 
 void SolverDDP::forwardPass(const double steplength) {
+  START_PROFILER("SolverDDP::forwardPass");
```

Comment:
Also BoxDDP solver here

Reply:
Same comment as above

Reply:
The forward passed for `FDDP`, `BoxDDP`, `BoxFDDP` are different. We don't inherit them.